pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -79,13 +79,6 @@ let
     qtserialport = [ ./qtserialport.patch ];
     qtwebengine = [
       # Fix build with bison-3.7: https://code.qt.io/cgit/qt/qtwebengine-chromium.git/commit/?id=1a53f599
```

Comment:
This comment needs to be removed as well

Reply:
removed